android/src/main/java/com/reactnativestripesdk/Mappers.kt

```diff
@@ -366,7 +366,7 @@ internal fun mapFromPaymentIntentResult(paymentIntent: PaymentIntent): WritableM
   paymentIntent.amount?.let {
     map.putDouble("amount", it.toDouble())
   }
-  paymentIntent.canceledAt?.let {
+  paymentIntent.canceledAt.let {
     map.putString("canceledAt", convertToUnixTimestamp(it))
   }
```

Comment:
If canceledAt is not null, you no longer need the let.
```suggestion
  map.putString("canceledAt", convertToUnixTimestamp(paymentIntent.canceledAt))
```